spec/system/trn_requests_spec.rb

```diff
@@ -372,9 +372,6 @@
 
   it "service is closed" do
     given_the_service_is_closed
-    and_i_visit_the_home_page
```

Comment:
Should these definitely be removed?

Reply:
You are right; the behaviour is correct as we are getting an auth error when the service is closed. Ive updated the spec to reflect that